assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -95,6 +100,23 @@ export default function SettingsView() {
 						<DisplaySetting value={ profileID } />
 					</p>
 				</div>
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						&nbsp;
+					</h5>
```

Comment:
Easiest and safest way to make the Links vertically aligned as per the AC (but this always feels a bit hacky!)

Reply:
This needs hiding on mobile (as per updated AC)

Reply:
We should make the links sibling elements of their respective `DisplaySetting`s instead. We want the link to be clearly associated with the value and putting it in a separate element like this spaces it in the grid like a separate thing. This also has the potential to wrap on smaller screens where it would lose the positional context as to what it is referring to. Keeping them in the same parent also has the benefit of not requiring a hack to preserve the alignment 👍

Reply:
Makes sense. Especially when going onto mobile

What do you think about the spacing? I just added one space.   


![same row](https://user-images.githubusercontent.com/8496063/127862296-f1767f53-d394-4259-b012-482641c29780.png)

Reply:
It does look a _little_ tight but also makes it more clear what the link refers to. I don't think we should get into adding `&nbsp;`s or another class to add space as this seems a bit arbitrary and makes the link a bit more ambiguous.

What do you think about this?

![image](https://user-images.githubusercontent.com/1621608/127880396-c35e1cfd-ab05-46c0-9c95-b0d764fa01d9.png)

This is how it looks if we also passed the `small` prop to the `Link`s? I realize the ACs specifically say that it should be the same size but I think it might look a bit better with a reduced size and we have an existing prop to do this. Seeing the options side-by-side would also help make the case for changing that in case we feel it looks better 🙂  We'd have to ask about changing it but I think prefer the smaller link. What do you think?

Reply:
Let's keep it as it is in the ACs for now, we can always follow-up with a super simple PR if we decide to tweak the Link size later.

Reply:
I don't think there's anything actionable here for me now @aaemnnosttv ?